src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsWatchLotButton.tsx

```diff
@@ -98,11 +100,6 @@ const ArtworkActionsWatchLotButton: FC<ArtworkActionsWatchLotButtonProps> = ({
   )
 }
 
-const UnfilledIcon = () => <BellIcon title="Unwatch lot icon" />
-const FilledIcon = () => (
-  <BellFillIcon title="Unwatch lot icon" fill="blue100" />
```

Comment:
This is super odd to me: it appears we add these titles for specs? But they are legitimate titles?

Firstly: [just don't use them](https://www.a11yproject.com/posts/title-attributes/). If you have an icon only button use an `aria-label` on the clickable node. And never name anything 'Foo icon'.